controllers/admin_client_mock.go

```diff
@@ -54,7 +54,7 @@ type mockAdminClient struct {
 	incorrectCommandLines                    map[string]bool
 	maxZoneFailuresWithoutLosingData         *int
 	maxZoneFailuresWithoutLosingAvailability *int
-	knobs                                    []string
+	knobs                                    map[string]struct{}
```

Comment:
I believe this one is actually a valid list, is there a reason to convert this into a map?

Reply:
FDB treats knobs as a set internally, right?  (I've never seen a knob that can be set multiple times, or examples of the knob parsing caring about the order of the knobs.)